presto-spi/src/main/java/com/facebook/presto/spi/ColumnHandle.java

```diff
@@ -13,6 +13,13 @@
  */
 package com.facebook.presto.spi;
 
+import com.facebook.presto.spi.type.NestedField;
+
+import java.util.Optional;
+
 public interface ColumnHandle
 {
```

Comment:
I will leave reviewing the api to the FB team, but this api needs to be redesigned I think.

I have two comments here:
- IIUC the `NestedField` will only be set for struct types, so it should be named properly to make this clear.
- I think of the `NestedField` here as a hint to the storage/file format to perform optimizations (if it supports), so one option to model these type of additional info is to add ability to attach storage hints to the column handles.